### PR TITLE
feat: Da eigen implementation docs & backup scripts

### PR DESCRIPTION
**Pull request by juan518munoz**
Docs on EigenDA integration, and scripts to bakcup and restore an ecosystem.

### CODE REVIEW

File: eigenda-integration.md [hunk 1]
```diff
@@ -71,3 +71,113 @@ zk_supervisor test integration --chain eigen_da
...
+## Holesky Setup
+
+### Modify localhost chain id number
+
+Modify line 32 in `zk_toolbox/crates/types/src/l1_network.rs`:
+
+```rs
+L1Network::Localhost => 17000,
+```
+
```
Comment (gianbelinche) [inline eigenda-integration.md:84]: This then needs a `./bin/zkt`

**Review by gianbelinche — CHANGES_REQUESTED** [review]
Also add the path fix for correct restoration

**Review by gianbelinche — APPROVED** [review]
LGTM

**Review by juanbono — APPROVED** [review]

File: eigenda-integration.md [hunk 2]
```diff
@@ -71,3 +71,119 @@ zk_supervisor test integration --chain eigen_da
...
+          --base-token-address 0x0000000000000000000000000000000000000001 \
+          --base-token-price-nominator 1 \
+          --base-token-price-denominator 1 \
+          --set-as-default false
+
+zk_inception ecosystem init \
+          --deploy-paymaster true \
+          --deploy-erc20 true \
+          --deploy-ecosystem true \
+          --l1-rpc-url https://ethereum-holesky-rpc.publicnode.com \
```
[the remark this answers is not in the document]
Comment (juan518munoz) [reply eigenda-integration.md]: Good point! Applied this change to the eigenDA proxy step too, as it also made use of this variable.

[the remark this answers is not in the document]
Comment (juanbono) [reply eigenda-integration.md]: great!

**Pull request merged**